File: src/main/java/br/com/cartola/metrics/repository/AtletaRepository.java
package br.com.cartola.metrics.repository;

import br.com.cartola.metrics.model.Atleta;
import org.springframework.data.mongodb.repository.MongoRepository;

/**
 * Created by samuel on 30/05/17.
 */
public interface AtletaRepository extends MongoRepository<Atleta, Integer> {
}

File: backend/src/main/java/br/com/cartola/metrics/repository/AtletaRepository.kt
package br.com.cartola.metrics.repository

import br.com.cartola.metrics.model.Atleta
import org.springframework.data.mongodb.repository.MongoRepository

/**
 * Created by samuel on 30/05/17.
 */
interface AtletaRepository : MongoRepository<Atleta, Int>
